### ISSUE TITLE
Repeating Section example is not working

### ISSUE THREAD

**Issue by michalkubat** [formly-js/ngx-formly#822]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
When you add a new investment by clicking the button "Add More Investment" and then edit any of the values, model is not updated.

**Minimal reproduction of the problem with instructions**
https://formly-js.github.io/ngx-formly/examples/advanced/repeating-section

**Comment by aitboudad** [comment]
duplicate of #821

**Comment by aitboudad** [comment]
fixed in `2.0.0-rc.44`
